src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDetailsMapping.java

```diff
@@ -244,11 +311,24 @@ public static PanelRequirements getPanelRequirements(SscsCaseData caseData) {
 
         panelRequirementsBuilder.panelPreferences(getPanelPreferences(caseData));
 
+        SessionCategoryMap sessionCategoryMap = getSessionCaseCode(caseData, referenceData);
+
         List<String> panelSpecialisms = new ArrayList<>();
-        // TODO Dependant on SSCS-10116 - Will be linked to PanelMemberSpecialism, need to find out how this is worked out
-        panelRequirementsBuilder.panelSpecialisms(panelSpecialisms);
+        if (nonNull(sessionCategoryMap)) {
+            String doctorSpecialism = caseData.getSscsIndustrialInjuriesData().getPanelDoctorSpecialism();
+            String doctorSpecialismSecond = caseData.getSscsIndustrialInjuriesData().getSecondPanelDoctorSpecialism();
+            for (PanelMember member : sessionCategoryMap.getCategory().getPanelMembers()) {
+                if (MQPM1.equals(member)) {
+                    panelSpecialisms.add(member.getReference(doctorSpecialism));
+                } else if (MQPM2.equals(member)) {
+                    panelSpecialisms.add(member.getReference(doctorSpecialismSecond));
+                }
+            }
+        }
```

Comment:
Needs breaking down with less nested.
Also jump out early if sessionCategoryMap is null